bluemix/terminal/table.go

```diff
@@ -157,3 +159,15 @@ func (t *PrintableTable) PrintJson() {
 	// mimic behavior of Print()
 	t.rows = [][]string{}
 }
+
+func (t *PrintableTable) PrintCsv() {
```

Comment:
I think we might want to have at least a couple of blocks of testing code on whether the error conditions are correct (e.g. some assertions on whether error output contains the text in lines 167/171 when the corresponding issues arise), to be better in-keeping with the other unit testing code. I can code this, or you can take a stab at it if you like @edsonarios.

Reply:
It actually seems that the nature of our `table` interface is appropriately restrictive enough (strings-only parameters to functions, only three exposed functions, etc.) that there shouldn't ever really be a `Write`/`WriteAll` error (which occurs when the specified buffer--in this case, the `table`'s writer--cannot be flushed). So, the testing seems adequate here.

I'd just add one scenario to test whether an empty table is correspondingly printed as a zero-length string (because the API responses that our code outputs are occasionally empty). You can add it to the test file changed, and I think this'll be ready for merge!

```
import (
        ...
        "strings"
)

func TestEmptyTable(t *testing.T) {
	buf := bytes.Buffer{}
	testTable := NewTable(&buf, []string{})
	testTable.PrintCsv()
	assert.Equal(t, len(strings.TrimSpace(buf.String())), 0)
}
```

Reply:
The unit test was added for empty table, however seeing the code it's possible to test these errors, but it will be necessary added a new function for change the delimiter that by defect is ',' for a personalized one (example: ';' ':'), so we can cause the error, setting an invalid character like '\n', and so we can test it

However, I don't know if this feature will be necessary or very useful.

But for now the unit test was added and it's working in the new output format

Reply:
I did think about changing the delimiter, but it seemed to me a case of going out of our way to think of an unrealistic scenario just to cause a parsing error. So, I agree with your comment that the usefulness of such a test is doubtful. I think we've covered our bases here.